src/main/java/Greet.java

```diff
@@ -0,0 +1,140 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Collections;
+
+public class Greet {
```

Comment:
Strange choice of class name. Since your 'Greet' object also does listing/deleting and saving